addon/utils/validators/shape.js

```diff
@@ -8,14 +8,15 @@ const {get, typeOf} = Ember
 import logger from '../logger'
 
 export default function (validators, ctx, name, value, def, logErrors, throwErrors) {
+  const shape = JSON.stringify(value, null, ' ')
```

Comment:
Oooh, I just realized `JSON.stringify()` can error sometimes, might need a try/catch around this.